corehq/apps/campaign/views.py

```diff
@@ -102,6 +102,16 @@ def _dashboard_gauge_configs(self, dashboard):
     def _get_gauge_config(dashboard_gauge):
         config = dashboard_gauge.to_widget()
         config['value'] = get_gauge_metric_value(dashboard_gauge)
+        # set max value of dial to nearest equivalent of 100
+        config['max_value'] = 10 ** len(str(config['value']))
+        config['major_ticks'] = [
+            '0',
+            int(0.2 * config['max_value']),
+            int(0.4 * config['max_value']),
+            int(0.6 * config['max_value']),
+            int(0.8 * config['max_value']),
+            config['max_value']
+        ]
```

Comment:
Nit: This could be make more concise with:
```suggestion
        config['major_ticks'] = ['0'] + [int(config['max_value'] * i/5) for i in range(1, 6)]
```

Reply:
Thanks @zandre-eng 

Did it slightly differently because I didn't need 0 to be integer,
https://github.com/dimagi/commcare-hq/pull/36065/commits/a0d1253087c3035da2b0acefc47ef9e1249375ed